client/src/containers/HomePage.tsx

```diff
@@ -117,56 +118,56 @@ class HomePage extends Component<HomePageProps, State> {
             )}
           </div>
           <div className="HomePage__samples">
-            <h5 className="text-center">... or view some sample portfolios:</h5>
+            <h5 className="text-center"><Trans>... or view some sample portfolios:</Trans></h5>
             <div className="container">
               <div className="columns">
                 <div className="column col-4 col-sm-12">
                   <div className="HomePage__sample">
                     <h6>
                       <Link to={this.state.sampleURLs[0]} onClick={() => {window.gtag('event', 'example-portfolio-1-homepage');}} >
-                        Kushner Companies / Westminster Management
+                        <Trans>Kushner Companies / Westminster Management</Trans>
                       </Link>
                     </h6>
                     <Link to={this.state.sampleURLs[0]} onClick={() => {window.gtag('event', 'example-portfolio-1-homepage');}} >
                       <img className="img-responsive" src={westminsterLogo} alt="Westminster" />
                     </Link>
-                    <p>
+                    <Trans render="p">
                       This property management company owned by the Kushner family is notorious for <a href="https://www.nytimes.com/2017/08/15/business/tenants-sue-kushner-companies-claiming-rent-rule-violations.html" target="_blank" rel="noopener noreferrer">violating rent regulations</a> and <a href="https://www.villagevoice.com/2017/01/12/jared-kushners-east-village-tenants-horrified-their-landlord-will-be-working-in-the-white-house/" target="_blank" rel="noopener noreferrer">harassing tenants</a>. The stake currently held by Jared Kushner and Ivanka Trump is worth as much as $761 million.
-                    </p>
-                    <Link className="btn block text-center" to={this.state.sampleURLs[0]} onClick={() => {window.gtag('event', 'example-portfolio-1-homepage');}} >View portfolio &#10230;</Link>
+                    </Trans>
+                    <Link className="btn block text-center" to={this.state.sampleURLs[0]} onClick={() => {window.gtag('event', 'example-portfolio-1-homepage');}} ><Trans>View portfolio</Trans> &#10230;</Link>
                   </div>
                 </div>
                 <div className="column col-4 col-sm-12">
                   <div className="HomePage__sample">
                     <h6>
                       <Link to={this.state.sampleURLs[1]}  onClick={() => {window.gtag('event', 'example-portfolio-2-homepage');}} >
-                        A&E Real Estate
+                        <Trans>A&amp;E Real Estate</Trans>
                       </Link>
                     </h6>
                     <Link to={this.state.sampleURLs[1]} onClick={() => {window.gtag('event', 'example-portfolio-2-homepage');}} >
-                      <img className="emassoc img-responsive" src={aeLogo} alt="E&M Associates" />
```

Comment:
Looks like this was a typo?  Was this originally an E&M Associates portfolio that got changes to A&E?

Reply:
oh yes, thanks for catching that! I changed that to A&E right before break